crates/table/src/table.rs

```diff
@@ -197,6 +201,7 @@ impl Table {
 
         // Insert the row into the page manager.
         let (hash, ptr) = self.insert_internal(blob_store, row)?;
+        self.row_count += 1;
```

Comment:
Should this happen inside `insert_internal`?

Reply:
you are right